test/index.js

```diff
@@ -109,6 +110,33 @@ tester.describe("scrape-it", t => {
             cb();
         });
     });
+    t.it("closet sample", cb => {
+        scrapeIt(URL, {
+            addresses: {
+                listItem: "table tbody tr",
+                data: {
+                    address: ".address",
+                    suburb: {
+                        selector: ".city",
+                        closest: "table",
+                        convert(html) {
+                            $ = cheerio.load(html);
```

Comment:
We can get rid of `cheerio` here, by passing the `$elm` variable as second argument: https://github.com/IonicaBizau/scrape-it/blob/810560e4729b6cc6a7414fa1871ff38bc6e9c143/lib/index.js#L167

It's going to be a performance improvement as well.